### PR TITLE
init push with feature engineering updates

### PR DESCRIPTION
**Pull request by kokomocha**
Updated scaler, correlation, and pca scripts according to pylint and pytest.s

### CODE REVIEW

File: config/data_init.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{   
+    "data_splitter":{
+        "ingest_path":"S:/NEU/Courses_Academic/IE7374_MLOps/Ecommerce-Data-MLOps/data/data.csv",
```
Comment (Thomas-George-T) [inline config/data_init.json:3]: Please rework this. Could be a potential issue later on

Comment (Thomas-George-T) [inline config/data_init.json]: Please rework this. Could be a potential issue later on

File: dags/data/Online Retail.xlsx
Comment (Thomas-George-T) [inline dags/data/Online Retail.xlsx:1]: Please remove this and recommit without this file. Airflow DAGs should be able to reconstruct this from our code.

File: dags/src/correlation.py
Comment (Thomas-George-T) [inline dags/src/correlation.py:1]: Please create one function for each module. You can create as many modules as required. This is okay for now.

**Review by Thomas-George-T — CHANGES_REQUESTED** [review]
Please remove the files under `dags/data` and `dags/processed`. They should be recreated whenever we run the Airflow DAGs. Please delete and recommit. 

Please add these files to `.gitignore` for future.

File: config/data_init.json (hunk 1 above)
Comment (kokomocha) [reply config/data_init.json]: Yes, I will remove it.

File: dags/data/Online Retail.xlsx
Comment (kokomocha) [reply dags/data/Online Retail.xlsx:1]: Okay

File: dags/src/correlation.py
Comment (kokomocha) [reply dags/src/correlation.py:1]: Yes, next scripts onwards.

**Review by Thomas-George-T — APPROVED** [review]
Everything seems fixed! Good job! Approved

**Pull request merged**